### PR TITLE
MODUSERS-193 move patron block functionality to mod-patron-blocks

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Purpose 
Remove patron block conditions and limits functionality due to moving it to mod-patron-blocks

Resolves [MODUSERS-193](https://issues.folio.org/browse/MODUSERS-193)

## Approach
- remove DB schema;
- remove init data;
- remove endpoints;

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -168,60 +168,6 @@
           "pathPattern" : "/_/tenant"
         }
       ]
-    },
-    {
-      "id": "patron-block-conditions",
-      "version": "0.1",
-      "handlers": [
-        {
-          "methods": ["GET"],
```
Comment (alexanderkurash) [inline descriptors/ModuleDescriptor-template.json:177]: Should we update module version when we remove interfaces from it?

File: src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -178,16 +178,6 @@
...
-    },
-    {
-      "tableName" : "patron_block_conditions",
-      "fromModuleVersion" : "mod-users-16.0.1",
-      "withMetadata" : true
-    },
-    {
-      "tableName" : "patron_block_limits",
-      "fromModuleVersion" : "mod-users-16.0.1",
-      "withMetadata" : true
```
Comment (alexanderkurash) [inline src/main/resources/templates/db_scripts/schema.json:190]: We should also delete tables

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (roman-barannyk) [reply descriptors/ModuleDescriptor-template.json:177]: Updated version of module to 16.2.1

Comment (roman-barannyk) [reply descriptors/ModuleDescriptor-template.json:177]: Updated version of module to 16.2.1

File: src/main/resources/templates/db_scripts/schema.json (hunk 2 above)
Comment (roman-barannyk) [reply src/main/resources/templates/db_scripts/schema.json:190]: done

**Pull request merged**